docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -38,84 +68,110 @@ Strapi currently support the following databases.
 | MariaDB    | 10.1            |
 | MongoDB    | 3.6             |
 
-## Step 2: Create a new project
+## Creating a Strapi project
 
-:::: tabs
+::: danger 🤓 CLI installation options
+The following installation guide covers the most basic installation option using the CLI. There are however other options that can be used when creating a new Strapi project:
 
-::: tab yarn
+- Using the `--quickstart` flag at the end of the command to directly create the project in quickstart mode.
+- Using the `--template` flag at the end of the command to create a project with pre-made Strapi configurations (see [Templates](templates.md)).
 
-To quickly create a blank project from scratch, run this command:
+Strapi also offers a starters CLI to create a project with a pre-made frontend application (see [our dedicated blog post](https://strapi.io/blog/announcing-the-strapi-starter-cli)).
+:::
 
-```bash
-yarn create strapi-app my-project --quickstart
-```
 
-Alternatively, to use one of our [starters](https://strapi.io/starters), run this command instead:
-
-```bash
-yarn create strapi-starter my-project <starter-url>
-```
+1. In a terminal, run the following command:
 
+:::: tabs card
+::: tab npm
+ ```sh
+  npx create-strapi-app my-project
+  ```
+::: 
+::: tab yarn
+ ```sh
+  yarn create strapi-app my-project
+ ```
 :::
 
-::: tab npx
+2. Choose an installation type:
+   * `Quickstart (recommended)`, which uses the default database (SQLite)
+   * `Custom (manual settings)`, which allows to choose your preferred database
 
-To quickly create a blank project from scratch, run this command:
+3. (Custom installation type only) Among the list of databases, choose a database for your Strapi project.
 
-```bash
-npx create-strapi-app my-project --quickstart
-```
+4. (Custom installation type only) Name your project's database.
+
+## Running Strapi
```

Comment:
Is it worth it to add a link somewhere to redirect to this documentation?
https://strapi.io/documentation/developer-docs/latest/developer-resources/cli/CLI.html#strapi-develop

Reply:
Could also be added in one-click guides maybe?

Reply:
Honestly I'm not sure because:
* I'm not sure the strapi-new part in the docs is up-to-date
* I have the feeling it would complicate the thing a bit (with the multiple parameters that can be passed)